.env

```diff
@@ -0,0 +1,3 @@
+PORT = 3000
+MONGO_URI = mongodb+srv://kapil1234:kapil1234@cluster0.xuhcbds.mongodb.net/codebcdb?retryWrites=true&w=majority
+JWT_SECRET_KEY = "secret key";
```

Comment:
we can use crypto library to generate unique random hash of large size and use it as secret key.